apps/system/js/app_chrome.js

```diff
@@ -319,7 +319,9 @@
   };
 
   AppChrome.prototype.titleClicked = function ac_titleClicked() {
-    if ((Service && Service.locked) || this.app.contextmenu.isVisible()) {
+    var contextMenu = this.app.contextmenu && this.app.contextmenu.isVisible();
+
+    if ((Service && Service.locked) || contextMenu) {
```

Comment:
nit: Since we broke contextMenu out of the if statement, can we do the same with the locked? So this just becomes something like: ```if (locked || contextMenu) {```